### ISSUE TITLE
[BUG] ListSlice(pad=True) does not generate valuecount.min and max properties in the schema

### ISSUE THREAD

**Issue by rnyak** [NVIDIA-Merlin/NVTabular#1749]
**Describe the bug**

When we add ListSlice(k, pad=True) op to a list column, it does not generate properties.value_count.min and properties.value_count.ma fields in the schema. These fields appear in the schema with `ValueCount()` op.

**Steps/Code to reproduce bug**

You can repro the issue with:

```
import cudf
import pandas as pd
import nvtabular as nvt
df = pd.DataFrame(
    {
        "user_id": [1, 3, 3, 4, 3, 1, 2, 4, 6, 7] *2,
        "item_id": [[1, 3, 10], [1, 3, 10], [4, 2, 1], [1, 2, 3], [1, 3, 10], [1, 3, 10], [1, 3, 10], [11, 3, 10], [11, 3, 10], [4, 2, 1]] *2,
    }
)

user_id = ["user_id"] >> Categorify() >> TagAsUserID()

item_id = ["item_id"] >> Categorify() >> TagAsItemID() >> ListSlice(5, pad=True)

train_dataset = nvt.Dataset(df)

workflow = nvt.Workflow(user_id + item_id)

workflow.fit(train_dataset)
workflow.transform(train_dataset).to_parquet("./train/")

# then run the following and check out the schema:
workflow.output_schema
```

**Expected behavior**

**Environment details (please complete the following information):**

- Environment location: [Bare-metal, Docker, Cloud(specify cloud provider)]

`merlin-tensorflow:22.12`  container with the latest main branches pulled.

**Comment by karlhigley** [comment]
What should happen in the case where `pad=False`?

**Comment by rnyak** [comment]
> What should happen in the case where `pad=False`?

@karlhigley  It returns the sequence without padding, and since in the example we set the ListSlice trimming value as `5` that is longer than the sequence lengths in each sample in the dataset, it returns the original length of the each sample. No trimming happens, and no value_count in the schema as well.

if we set the `ListSlice(2, pad=False)` then the lists are trimmed to length of 2.

**Comment by rnyak** [comment]
> In that case, I think we know that the maximum length is no more than 5 though, right? So we can set an upper bound on the dimension of the shape that represents list length (currently captured in the `value_counts`.)

In the example above, all the list entries have length of 3. So we know the ListSlice(5) > 3, so it wont do any trimming. And yes, if we can capture the max_length of the longest list in the given train (input) dataset, we can set an upper bound on the dim of shape.

**Comment by rnyak** [comment]
@karlhigley is there any update for this ticket? or any fixes going on? thanks.

**Comment by rnyak** [comment]
> AFAIK this has been fixed for a while now by the changes in #1760

@karlhigley it worked for me.. is #1760 part of `23.02` release?

**Comment by karlhigley** [comment]
Should be, yes!
